elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidator.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers.validator;
+
+import com.yahoo.elide.contrib.dynamicconfighelpers.DynamicConfigHelpers;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideSecurityConfig;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideTableConfig;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.Table;
+
+import org.apache.commons.io.FileUtils;
+
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Slf4j
+@Data
+public class DynamicConfigValidator {
+
+    private static ElideTableConfig elideTableConfig;
+    private static ElideSecurityConfig elideSecurityConfig;
+    private static Map<String, Object> variables;
+    private static final String[] SQL_DISALLOWED_WORDS = new String[] { "DROP ", "TRUNCATE ", "DELETE ", "INSERT ",
```

Comment:
There are more DDL and DCL commands than this:
ALTER, COMMENT, CREATE, DESCRIBE, SHOW, USE, GRANT, REVOKE, CONNECT, LOCK, EXPLAIN, CALL, MERGE, RENAME

Reply:
We should also add a semicolon to the list.

Reply:
Added additional keywords.
Added explicit check that sql should not contain semicolon.
Changed to split SQL on whitespaces and match exact keywords now